cs/src/core/Clone.cs

```diff
@@ -37,30 +37,58 @@ public static T From<SourceT>(SourceT source)
     public class Cloner<SourceT>
     {
         readonly Func<object, object>[] clone;
-        
+
+        /// <summary>
+        /// Create a cloner that makes clones of the same type SourceT as source objects.
+        /// </summary>
+        public Cloner()
+            : this(typeof(SourceT))
+        {}
+
         /// <summary>
         /// Create a cloner that makes clones of the specified type.
         /// </summary>
         /// <param name="type">type of clone object, may be different than source object</param>
         public Cloner(Type type)
+            : this(type, (IParser) null)
+        {}
+
+        /// <summary>
+        /// Create a cloner that makes clones of the specified type.
+        /// </summary>
+        /// <param name="type">type of clone object, may be different than source object</param>
+        /// <param name="parser">Custom <see cref="IParser"/> instance</param>
+        public Cloner(Type type, IParser parser)
         {
-            clone = Generate(type, new DeserializerTransform<object>(
-                (o, i) => clone[i](o)));
+            clone = Generate(type,
+                             new DeserializerTransform<object>((o, i) => clone[i](o)),
+                             parser);
         }
 
         /// <summary>
+        /// Create a cloner that makes clones of the specified type.
+        /// </summary>
+        /// <param name="type">type of clone object, may be different than source object</param>
+        /// /// <param name="factory">factory implementing IFactory interface</param>
```

Comment:
Minor: Let's use the `<see>` tag for refering to `IParser` and `IFactory`.
This needs to be done elsewhere in the file as well.